echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/googlechat/GoogleChatMessage.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2018 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.echo.googlechat;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+public class GoogleChatMessage {
+  transient static String SPINNAKER_ICON_URL = "https://avatars0.githubusercontent.com/u/7634182?s=200&v=4";
```

Comment:
Can we make these configurable? Or at least add a todo? I know some companies reskin Spinnaker internally.

Reply:
TODO added.

https://github.com/spinnaker/echo/pull/282/files/5550d3cd314a0692ead8a3e25c19571160932d9c#diff-cf2fcc14aa053805faa519b286678142R24